### DISCUSSION TITLE
[11주차] 온라인 스터디

### DISCUSSION THREAD

**Discussion by ChaerinYu** [no-study-no-future/interview-study#80] [General]
~ 20:15 각자 PR리뷰
~ 21:00 발표
~ 21:10 회고 (좋았던 점, 건의사항, 느낀점 등)

**Comment by ChaerinYu** [comment]
### 발표
1. 유채린
 * [ETC/DesignPattern] Builder Pattern #78
2. 김정윤
 * [Network] JWT #75
 3. 이동호
* [Algorithm] Binary search #79

**Comment by ChaerinYu** [comment]
### 회고 (a.k.a. 출석체크)
* 참석 확인용이므로 `감사합니다!` 만 작성해주셔도 좋아요 😊
* 좋았던 점
* 건의사항
* 느낀점

**Reply by KJY97** [reply]
감사합니다!

**Reply by doho-ho** [reply]
감사하빈다!

**Reply by ChaerinYu** [reply]
정윤님 감사해요~~!
